addons/mail/static/src/owl/components/composer/composer.scss

```diff
@@ -149,42 +166,56 @@
                 }
             }
         }
+    } }
 
-    }
+.o_Composer_button.o-open {
+    background-color: gray('200');
 }
 
-.o_Composer_button {
-
-    &:focus {
-        box-shadow: none;
-        background-color: gray('200');
-    }
+.o_Composer_buttonDiscard {
+    border: 1px solid lighten(gray('400'), 5%);
 
-    &.o-open {
-        background-color: gray('200');
+    &.o_Composer_buttonInline {
+        border-left: none;
     }
 }
 
-.o_Composer_buttonDiscard {
-    border: 1px solid lighten(gray('400'), 5%);
-    border-left: none;
+.o_Composer_buttons:not(.o-composer-is-compact) {
+    border: 0;
+
+    .o_Composer_toolButtons{
```

Comment:
- missing whitespace before `{`
- avoid CSS specificity, use a modifier on this class instead